portfolio/src/main/webapp/script.js

```diff
@@ -119,3 +119,21 @@ function togglePause() {
     paused = true;
   }
 }
+
+// function that fetches data from the /data URL
+// and displays it on the page
+function loadComments () {
+    fetch('/data').then(response => response.json()).then(comments => {
+        const commentList = document.getElementById('comments');
+        commentList.innerHTML = '';
```

Comment:
A better way to clear the node is to remove the child one-by-one. (It might be better to start from lastChild since it is an array, removing from first have O(n) complexity, but I don't know if v8 already optimize that)

Setting innerHTML is in general discouraged for security reasons. It might have other consequences but I don't remember exactly :)